bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -942,10 +944,8 @@ private void searchSmartHomeDevicesRecursive(@Nullable Object jsonNode, List<Obj
             if (map.containsKey("entityId") && map.containsKey("friendlyName") && map.containsKey("actions")) {
                 // device node found, create type element and add it to the results
                 JsonElement element = gson.toJsonTree(jsonNode);
-                if (element.getAsJsonObject().get("applianceId").getAsString().contains("AAA")) {
```

Comment:
The check for AAA looks strange for me :) So I have removed it. What was your intention?

Reply:
All of the lights or plugs start with AAA so I used it to select the devices without the openhab skill devices.
This condition could be changed so that we are able to integrate the current openhab devices from the skill.

Reply:
My idea is, that we can control devices from other skills too, only devices of OH should be suppressed, so I removed the check here, and filter the devices in the SmartHomeDeviceDiscovery